build/Dockerfile

```diff
@@ -71,23 +69,23 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # TODO(tamird): use the shallow clone when crosstool-ng 1.23 is released.
 # RUN git clone --depth 1 --branch crosstool-ng-1.23.0 https://github.com/crosstool-ng/crosstool-ng \
-RUN git clone https://github.com/crosstool-ng/crosstool-ng && git -C crosstool-ng checkout 1ec5e7aa3400b534264ba1c2fa3949dc6a2cca90 \
+RUN git clone https://github.com/crosstool-ng/crosstool-ng && git -C crosstool-ng checkout ee7662fa58a7f45ccd3e3520919ee70301ab7648 \
  && cd crosstool-ng \
  && ./bootstrap \
  && ./configure --prefix /usr/local/ct-ng \
  && make -j$(nproc) \
  && make install \
+ && cp ct-ng.comp /etc/bash_completion.d/ \
  && cd .. \
  && rm -rf crosstool-ng
 
-COPY x86-linux-gnu.defconfig x86-linux-musl.defconfig ./
-RUN mkdir build && cd build \
- && rm -rf * && cp ../x86-linux-gnu.defconfig  defconfig && /usr/local/ct-ng/bin/ct-ng defconfig && CC=gcc CXX=g++ /usr/local/ct-ng/bin/ct-ng build \
- && rm -rf * && cp ../x86-linux-musl.defconfig defconfig && /usr/local/ct-ng/bin/ct-ng defconfig && CC=gcc CXX=g++ /usr/local/ct-ng/bin/ct-ng build \
+COPY x86_64-unknown-linux-gnu.defconfig x86_64-unknown-linux-musl.defconfig x86_64-w64-mingw.defconfig ./
+RUN mkdir build src && cd build \
+ && rm -rf * && cp ../x86_64-unknown-linux-gnu.defconfig  defconfig && /usr/local/ct-ng/bin/ct-ng defconfig && /usr/local/ct-ng/bin/ct-ng build \
+ && rm -rf * && cp ../x86_64-unknown-linux-musl.defconfig defconfig && /usr/local/ct-ng/bin/ct-ng defconfig && /usr/local/ct-ng/bin/ct-ng build \
+ && rm -rf * && cp ../x86_64-w64-mingw.defconfig          defconfig && /usr/local/ct-ng/bin/ct-ng defconfig && /usr/local/ct-ng/bin/ct-ng build \
```

Comment:
I feel dumb - what is happening here? Why are you copying stuff to `.` and deleting it over and over again? Can you add a comment at least?